### PR TITLE
Daily reporter

### PR DESCRIPTION
**Pull request by chrismaree**

### CODE REVIEW

File: reporters/SponsorReporter.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  constructor(expiringMultiPartyClient, walletsToMonitor, priceFeed) {
+    this.empClient = expiringMultiPartyClient;
+    this.walletsToMonitor = walletsToMonitor;
+    this.priceFeed = priceFeed;
+
+    this.web3 = this.empClient.web3;
+
+    this.formatDecimalString = createFormatFunction(this.web3, 2);
+
+    this.collateralCurrencySymbol = "DAI";
```
Comment (nicholaspai) [inline reporters/SponsorReporter.js:16]: You could grab these from the `empClient.emp` right?

File: reporters/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const { TokenBalanceClient } = require("../financial-templates-lib/clients/TokenBalanceClient");
+
+const { SponsorReporter } = require("./SponsorReporter");
+
+// Truffle contracts
+const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+const ExpandedERC20 = artifacts.require("ExpandedERC20");
+
+async function run(address, walletsToMonitor, priceFeedConfig) {
+  console.log(`Starting Reporter Script🖨\n EMP Address: ${address}`);
```
Comment (nicholaspai) [inline reporters/index.js:24]: nice emoji

File: reporters/SponsorReporter.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+const chalkPipe = require("chalk-pipe");
+const blue = chalkPipe("blue");
+const bold = chalkPipe("bold");
+
+class SponsorReporter {
+  constructor(expiringMultiPartyClient, walletsToMonitor, priceFeed) {
+    this.empClient = expiringMultiPartyClient;
+    this.walletsToMonitor = walletsToMonitor;
```
Comment (nicholaspai) [inline reporters/SponsorReporter.js:9]: What's the example JSON shape of `walletsToMonitor`? Worth documenting somewhere

Comment (chrismaree) [reply reporters/SponsorReporter.js:9]: for sure. right now there is no docs or comment in this initial stub.

File: reporters/SponsorReporter.js (hunk 1 above)
Comment (chrismaree) [reply reporters/SponsorReporter.js:16]: right now no. it's actually not sored there, just like in the other bots we actually don't have this info. it's an outstanding Toto to improve.

**Pull request merged**